backend/controller/sql/queries.sql

```diff
@@ -717,7 +717,8 @@ WITH cursor AS (
 )
 SELECT events."key" as event,
         events.payload,
-        events.created_at
+        events.created_at,
+        (EXTRACT(EPOCH FROM (NOW() - events.created_at))::double precision >= sqlc.arg('consumption_delay')) AS ready
```

Comment:
You can just send a duration in directly, and it will vastly simplify the query.

Reply:
🤦‍♂️ simplified it